packages/github-lens-api/src/app.ts

```diff
@@ -7,7 +7,7 @@ import { getRouteDescriptions } from './controller/DescribeApiController';
 import { getHealthCheckHandler } from './controller/HealthCheckController';
 import { getMembers, getMembersByLogin } from './controller/MembersController';
 import { getAllRepos, getRepoByName } from './controller/ReposController';
```

Comment:
Do these need updating too?

```suggestion
import { getHealthCheckHandler } from './controller/healthcheck';
import { getMembers, getMembersByLogin } from './controller/members';
import { getAllRepos, getRepoByName } from './controller/repos';
```

Reply:
👍 coding on the bus & relying on the IDE for refactor are not great working practises.